sources/packages/backend/workflow/test/test-utils/factories/student-dependent.ts

```diff
@@ -33,6 +33,17 @@ export enum DependentChildCareEligibility {
   Eligible12YearsAndOver = "Eligible12YearsAndOver",
 }
 
+export enum DependentCSGDEligibility {
```

Comment:
Please do not create a new enum; expand the existing one with more options, if required.
The idea is to create something generic and not coupled to any specific grant.